sql/async_udfs.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Asynchronous User-Defined Functions
+description: 'Run Async UDFs'
+---
+
+Arroyo's UDFs can be asynchronous.
+This is useful for UDFs that need to make network requests or perform other asynchronous operations.
```

Comment:
It's maybe worth emphasizing that the distinction is that normal UDFs need to run very quickly (<<1ms) because they block processing, while asynchronous UDFs can run in parallel and can therefore take much longer to execute while still maintaining good throughput.

I would also include the caveat that async UDFs cannot block—they must use async operations or use spawn_blocking (https://docs.rs/tokio/latest/tokio/task/fn.spawn_blocking.html)

Reply:
Yup, done.